grafana-plugin/src/models/alertgroup/alertgroup.types.ts

```diff
@@ -41,6 +43,11 @@ export interface GroupedAlert {
   render_for_web: RenderForWeb;
 }
 
+export type AlertGroup = components['schemas']['AlertGroup'] & {
+  loading?: boolean;
```

Comment:
just for my own curiosity, where is this `loading` attribute used? Based on the name I'm guessing it's for... some loading state indicator? If so, could that state be managed external from the response objects we get back from the API? (would get rid of the need to have this type if I understand correctly)

Reply:
You're right and yes, our plan on the frontend is to move every loading state like that to a separate dedicated store. So that they are completely separated from models/domains themselves